Doc/whatsnew/3.6.rst

```diff
@@ -2052,6 +2052,8 @@ tkinter
 The :mod:`tkinter.tix` module is now deprecated.  :mod:`tkinter` users
 should use :mod:`tkinter.ttk` instead.
 
+.. _whatsnew-3.6-venv:
```

Comment:
Sidenote: Following the convention I've used for the 3.11 What's New would be `whatsnew36-venv`, but I think this convention (or at least `whatsnew3.6-venv`) would be better—out of old habit I often avoid using `.` in ref target labels. In any case, no practical reason to change it here.

Reply:
Sorry for the confusion—I was mentioning this as a sidenote and suggestion you _not_ change it here (instead, I really should be using it in the ref targets I'm adding, though probably too late now for 3.11—I can adopt it with 3.12). But since you have changed it, not really worth changing it again.

Just to be more clear for the future, if I'm actually requesting a change, I will pretty much always make it as a actual suggestion (unless its impractical or impossible), so you can easily apply it with `Files` -> `Add to batch` -> `Commit` rather than manually implementing, committing, pushing and resolving it. The way I see it is if I'm going to ask someone else to make a change on their PR (unless its really important), its only fair that I put in the work myself to implement it, so all they have to do is click a button to accept it.

Reply:
I see I see, don't worry too much about it - I checked the existing labels and just decided that it would be good to follow convention. And conversely, I requested the review so I definitely don't mind the work (and I would have to run it locally anyway just to check the rendering), but of course the ease of changing is always appreciated.